### PR TITLE
[SHARE-393][Feature] Zenodo oai_datacite3 Harvester/Normalizer

### PR DESCRIPTION
**Pull request by leb2dg**
Continuing #354 Related to #308 
### Changes
- update Datacite normalizer to check for `resourceType = preprint`
- use Datacite normalizer for Zenodo
- new harvester under `zenodo/datacite/`
- disable oai harvester

### CODE REVIEW

**Comment by leb2dg** [comment]
@chrisseto, this runs through the whole system fine 👍

File: providers/org/datacite/normalizer.py [hunk 1]
```diff
@@ -232,6 +233,21 @@ class ThroughTags(Parser):
 
 class CreativeWork(Parser):
 
+    @property
+    def schema(self):
+        try:
+            resourceType = self.context['record']['metadata']['oai_datacite']['payload']['resource']['resourceType']['#text']
+            print(resourceType)
```
Comment (chrisseto) [inline providers/org/datacite/normalizer.py]: Left over print statement

File: providers/org/datacite/normalizer.py [hunk 2]
```diff
@@ -584,3 +600,10 @@ def list_in_string(self, string, list_):
                 if word.search(string):
                     return True
         return False
+
+
+class DataciteNormalizer(Normalizer):
+
+    def do_normalize(self, data):
```
Comment (chrisseto) [inline providers/org/datacite/normalizer.py]: You shouldn't have to define this method.

File: providers/org/zenodo/oai/apps.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from share.provider import OAIProviderAppConfig
+
+
+class AppConfig(OAIProviderAppConfig):
```
Comment (chrisseto) [inline providers/org/zenodo/oai/apps.py]: Please override the user property like `org.arxiv.oai`

File: providers/org/datacite/normalizer.py (hunk 2 above)
Comment (leb2dg) [inline providers/org/datacite/normalizer.py]: @chrisseto, so just call `super`? I need something there so I can import the class right?

Comment (chrisseto) [inline providers/org/datacite/normalizer.py]: You can just define the class without anything else.
```python
class DataciteNormalizer(Normalizer):
    pass
```

**Comment by chrisseto** [comment]
Seems that Zenodo's OAI endpoint does not like `oai_datacite3` anymore. Waiting on a response from them.

**Comment by chrisseto** [comment]
@leb2dg Would you mind rebasing this onto develop? Let me know if you'd like any help, Looks like things got a touch messy. (Sorry)

**Comment by leb2dg** [comment]
@chrisseto :1st_place_medal:

**Comment by chrisseto** [comment]
@leb2dg can you set this up like `com.figshare.v2`?

Leave `org.zenodo` as is and create the new one as `org.zenodo.datacite` (Or something) then add a migration that disables the original `org.zenodo` harvester? 

I won't have to migrate and data that way 👍 

Otherwise LGTM

File: providers/org/zenodo/migrations/0001_initial.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.9.7 on 2016-07-08 15:45
+# Generated by Django 1.9.7 on 2016-08-09 19:28
```
Comment (chrisseto) [inline providers/org/zenodo/migrations/0001_initial.py]: Can you revert this line?

**Pull request merged**